contracts/ts/genMaciState.ts

```diff
@@ -130,14 +130,12 @@ export const genMaciStateFromContract = async (
 
   const treeDepths = {
     intStateTreeDepth: Number(onChainTreeDepths.intStateTreeDepth),
-    messageTreeDepth: Number(onChainTreeDepths.messageTreeDepth),
-    messageTreeSubDepth: Number(onChainTreeDepths.messageTreeSubDepth),
     voteOptionTreeDepth: Number(onChainTreeDepths.voteOptionTreeDepth),
   };
 
   const batchSizes = {
     tallyBatchSize: STATE_TREE_ARITY ** Number(onChainTreeDepths.intStateTreeDepth),
-    messageBatchSize: MESSAGE_TREE_ARITY ** Number(onChainTreeDepths.messageTreeSubDepth),
+    messageBatchSize: MESSAGE_BATCH_SIZE,
```

Comment:
this should not be a constant but come from the poll contract

Reply:
resolved